app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt

```diff
@@ -62,7 +54,7 @@ fun OutOfSpaceAppItem(packageName: String) {
     AppItem(
       icon = appIcon,
       name = app.name,
-      appSize = app.appSize,
+      versionName = app.versionName,
```

Comment:
Having version name instead of the app size is probably an error. Check with design if it should be like this